src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -486,9 +493,18 @@ const ExpandAggregateRow = (props: {
   return <React.Fragment>{children({willExpand: false})}</React.Fragment>;
 };
 
-const StyledIconStack = styled(InlineSvg)`
-  vertical-align: top;
-  color: ${p => p.theme.gray3};
+const HeaderIcon = styled('span')`
+  & > svg {
```

Comment:
Should we use component as a selector here?

Reply:
Can't, it's not a styled component.

Reply:
Dang

Reply:
And it could be different components based on the conditional logic.